app/views/movie/top_forty.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Discover Top Movies</h1>
```

Comment:
Love this! If we change the controller action to index, this might be our /movies index.html.erb page eventually :) The controller would hold the logic about what to show in the view (top40 or search), the view would just show results